### DISCUSSION TITLE
Inconsistent ammo type naming

### DISCUSSION THREAD

**Discussion by A2ur3** [FakeFishGames/Barotrauma#14944] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

#13704
<entityname.physicoriumammobox>Physicorium Ammunition Box</entityname.physicoriumammobox>
<entityname.nuclearshellcheap>Nuclear Shell (cheap)</entityname.nuclearshellcheap>

### Reproduction steps

.

### Bug prevalence

Just once

### Single player or multiplayer?

Single player

### Version

v1.6.17.0 (Unto the Breach Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by JeroonBosch** [comment]
Oops. Anyways, will be addressed in the next update, closing.
